pkg/manifests/manifests.go

```diff
@@ -1769,6 +1783,14 @@ func (f *Factory) PrometheusAdapterDeployment(apiAuthSecretName string, requesth
 		},
 	)
 
+	if auditLogsEnabled(config.AuditLevel) {
+		lvl := strings.ToLower(config.AuditLevel)
```

Comment:
@PhilipGough continuing from the previous comment ... 

providing a `valid` log level other than `none` adds `--audit-` flags to the `prometheus adapter`